src/content/effective-dart/design.md

```diff
@@ -621,41 +635,23 @@ doesn't do that, it's best to assume you should *not* extend the class.
 Otherwise, later changes to it may break your code.
 
 
-### DO document if your class supports being extended
-
-This is the corollary to the above rule. If you want to allow subclasses of your
-class, state that. Suffix the class name with `Base`, or mention it in the
-class's doc comment.
-
-
-### AVOID implementing a class that isn't intended to be an interface
-
-Implicit interfaces are a powerful tool in Dart to avoid having to repeat the
-contract of a class when it can be trivially inferred from the signatures of an
-implementation of that contract.
-
-But implementing a class's interface is a very tight coupling to that class. It
-means virtually *any* change to the class whose interface you are implementing
-will break your implementation. For example, adding a new member to a class is
-usually a safe, non-breaking change. But if you are implementing that class's
-interface, now your class has a static error because it lacks an implementation
-of that new method.
+### DO use class modifiers to control if your class can be extended
 
-Library maintainers need the ability to evolve existing classes without breaking
-users. If you treat every class like it exposes an interface that users are free
-to implement, then changing those classes becomes very difficult. That
-difficulty in turn means the libraries you rely on are slower to grow and adapt
-to new needs.
+Since Dart 3.0, class modifiers like `final`, `base`, or `sealed` 
```

Comment:
```suggestion
Class modifiers like `final`, `base`, or `sealed` 
```

Dart 3 has been out for a while. I think we can avoid mentioning it. @johnpryan, your thoughts?

Reply:
I agree - we can leave this out here.